docs/ui/auth/fragments/web/authenticator.md

```diff
@@ -1068,6 +1068,35 @@ To achieve this, you first need to setup the userpool to allow email or phone nu
 
 </docs-filter>
 
+### Override Sign Up
+
+You can override the call to [`signUp`](~/lib/auth/emailpassword.md) function with `handleSignUp` property. For example, you can add a [custom attribute](~/lib/auth/emailpassword.md) like so:
```

Comment:
I tried using 
```
[custom attribute](~/lib/auth/emailpassword.md#custom-attributes)
```
to link to exact subsection, but the page wasn't navigating to the correct subsection so I just linked it to `~/lib/auth/emailpassword.md` for now.

Reply:
![Kapture 2021-06-24 at 17 41 46](https://user-images.githubusercontent.com/43682783/123498480-faaf8a00-d5e4-11eb-8fca-af26cb1aaf63.gif)
Maybe this is a bug or I'm missing something? cc @tjleing

Reply:
Talked offline, found that
- opening a new tab works
- looks like the docs is trying to scroll to the original height before navigation

and will be investigated.

This is not blocking this PR, so we'll go ahead and get this reviewed / merged.